CHANGELOG.md

```diff
@@ -11,6 +11,32 @@ Some experimental features can be enabled / disabled using [Feature Gates](docs/
 
 ## Unreleased
 
+## 0.10.2 - 2020-11-10
+
+### Changed
+
+- When the destination is a Service in a Traceflow request, automatically set the TCP SYN flag so the packet can be processed by AntreaProxy correctly. ([#1386](https://github.com/vmware-tanzu/antrea/pull/1386) [#1378](https://github.com/vmware-tanzu/antrea/pull/1378), [@lzhecheng] [@mengdie-song])
```

Comment:
Is it a fix?

Reply:
I moved it to fixes. I think since there was no way to set the SYN flag from the Octant UI, we can consider that it was a bug